src/SolidStateDetector/Semiconductor.jl

```diff
@@ -27,17 +19,19 @@ function Semiconductor{T}(dict::Dict, input_units::NamedTuple, transformations::
     else
         ConstantImpurityDensity{T}(0)
     end
-    sc.charge_drift_model = if haskey(dict, "charge_drift_model") && haskey(dict["charge_drift_model"], "model")
+    charge_drift_model = if haskey(dict, "charge_drift_model") && haskey(dict["charge_drift_model"], "model")
         cdm = getfield(Main, Symbol(dict["charge_drift_model"]["model"])){T}
         cdm <: AbstractChargeDriftModel{T} ? cdm() : ElectricFieldChargeDriftModel{T}()
     else
         ElectricFieldChargeDriftModel{T}()
     end
-    sc.material = material_properties[materials[dict["material"]]]
-    sc.geometry = transform(Geometry(T, dict["geometry"], input_units), transformations)
-    sc.geometry_positive, sc.geometry_negative = get_decomposed_volumes(sc.geometry)
-    sc.decomposed_surfaces = vcat(get_decomposed_surfaces.(sc.geometry_positive)...)
-    return sc
+    temperature = haskey(dict, "temperature") ? T(dict["temperature"]) : T(80)
```

Comment:
This 80K is arbitrary... The config files for germanium are all based on 78K as reference temperature.

Reply:
Hmm. That's for germanium. In general I actually thinking about setting it to zero or room temperature? :thinking:

Reply:
Well, I don't think anyone will operate their detectors at 0K, so change it to LN temperature or room temperature.

Reply:
Okay, I changed it (https://github.com/JuliaPhysics/SolidStateDetectors.jl/pull/163/commits/9e6f55a215cc912f8b2e9ee953ba5379db246599) in general to room temperature and if the material is HPGe the default is 78K.